### DISCUSSION TITLE
Portuguese (European) Translation

### DISCUSSION THREAD

**Discussion by marcop93** [ChristopherKlay/StadiaEnhanced#91] [Translations]
Language code: pt

```
"default": "Padrão",
"native": "Nativo",
"hide": "Esconder",
"show": "Mostrar",
"visible": "Visivel",
"hidden": "Escondido",
"enabled": "Activado",
"disabled": "Desactivado",
"auto": "Automático",
"manual": "Manual",
"games": "Jogos",
"bundles": "Bundles",
"addons": "Add-ons",
"responsive": "Responsivo",
"windowed": "Modo Janela",
"fullscreen": "Ecrã completo",
"searchstore": "Pesquisar na loja",
"onsale": "Em promoção",
"prodeals": "Promoções Pro",
"allgames": "Todos os jogos",
"usermedia": "Screenshots & Videos",
"searchbtnbase": "Pesquisar em",
"avatarpopup": "Novo URL para avatar (vazio para o padrão):",
"searchheader": "Incluindo Jogos",
"sessiontime": "Tempo de sessão",
"codec": "Codec",
"resolution": "Resolução",
"hardware": "Hardware",
"software": "Software",
"trafficsession": "Tráfego da sessão",
"trafficcurrent": "Tráfego instantâneo",
"trafficaverage": "Tráfego médio",
"packetloss": "Pacotes perdidos",
"framedrop": "Frames perdidos",
"latency": "Latencia",
"jitter": "Buffer de Jitter",
"decodetime": "Tempo de descodificação",
"compression": "Compressão",
"streammon": "Monitor do Stream",
"stream": "Stream",
"community": "Comunidade",
"speedtest": "Teste de velocidade",
"quickaccess": "Acesso Rápido",
"messages": "Mensagens",
"avatar": "Avatar",
"interface": "Interface",
"shortcut": "Atalhos",
"shortcuttitle": "Instalar atalho para",
"shortcutdesc": "Permite-te instalar um atalho para um jogo no teu dispositivo",
"gridsize": "Tamanho da Grelha",
"griddesc": "Muda o número de jogos por linha no ecrã inicial.",
"clock": "Relógio",
"clockdesc": "Mostra a hora actual na lista de amigos, em sobreposição no jogo, ou ambos.",
"friendslist": "Lista de Amigos",
"igoverlay": "Sobreposição no jogo",
"listoverlay": "Lista e Sobreposição",
"filter": "Filtro",
"filterdesc": "Permite-te organizar o teu ecrã inicial escondendo jogos. O filtro pode ser alternado pelo simbolo no topo direito, acima dos teus jogos no ecrã inicial.",
"filtertoggle": "Alternar",
"filterquick": "Rápido",
"invitebase": "Copiar ligação de convite",
"inviteactive": "Copiado!",
"prolabel": "Etiqueta Pro",
"prolabeldesc": "Remove a etiqueta 'Pro' dos jogos no ecrã inicial.",
"homegallery": "Galeria do Utilizador",
"homegallerydesc": "Esconde a área 'Capturas' no fundo do ecrã inicial.",
"quickprev": "Pré-visualização de mensagens",
"quickprevdesc": "Esconde a pré-visualização de mensagens na tua lista de amigos.",
"quickrep": "Resposta Rápida",
"quickrepdesc": "Esconde a opção de resposta rápida nas conversas.",
"offlinefriend": "Amigos Offline",
"offlinefrienddesc": "Esconde amigos offline na lista de amigos.",
"invisiblefriend": "Amigos Invisíveis",
"invisiblefrienddesc": "Esconde amigos com estado desconhecido na lista de amigos.",
"streammode": "Modo de Streaming",
"streammodedesc": "Torna certos elementos (p.e. a lista de amigos) não legíveis enquanto estás a fazer streaming (via ferramentas como OBS / Discord).",
"catprev": "Pré-visualização da Categoria",
"catprevdesc": "Esconde a etiqueta da categoria ao passar por cima de um jogo.",
"popup": "Efeito Popup",
"popupdesc": "Remove o efeito de zoom-in / aumento ao passar por cima de um jogo no ecrã inicial.",
"streammondesc": "Activa para iniciar o monitor quando um jogo começa.",
"resolutiondesc": "A resolução pretendida para stream de jogos. 1440p e 2160p requerem VP9.",
"codecdesc": "O codec utilizado para stream de jogos.",
"confirmreset": "De certeza que queres reiniciar as configurações?",
"gamesfinished": "Jogo terminado",
"achievementsunlocked": "Conquistas desbloqueadas",
"splitstore": "Dividir Listas da Loja",
"splitstoredesc": "Divide as listas da loja em duas colunas para uma melhor visão geral.",
"scrollbar": "Scrollbar",
"scrollbardesc": "Activa as scrollbars no ecrã inicial e loja.",
"resetsettings": "Reiniciar Configurações"
```

**Comment by ChristopherKlay** [comment]
Will be added in `1.8.6`.

Thank you for the work (:

**Reply by Gabsletobar** [reply]
update please:

"wishlist": "Lista de desejos",
"userprofile": "Meu Perfil",
"inlineimage": "Prévia de imagem",
"inlinedesc": "Substitui links de imagem para formatos comuns de ficheiros (jpg/gif/png) com uma prévia clicável.",
"familyelements": "Opções de partilha de família",
"familyelementsdesc": "Esconder a opção 'Partilhar este jogo com a família.'",

**Reply by ChristopherKlay** [reply]
Will be included in the next version releasing soon (:

Thank you!

**Reply by Gabsletobar** [reply]
Here's another one:
Version 1.10.x

"total": "Total",
"gamelabel": "Etiquetas de Jogos",
"gamelabeldesc": "Remove as etiquetas como por exemplo 'Pro' dos jogos da página inicial.",

**Comment by Gabsletobar** [comment]
Hello! I updated the translation for **V.1.11.X(2)**
Keep the amazing work for the extension :)
Greetings!

                default: 'Padrão',
                native: 'Nativo',
                hide: 'Esconder',
                show: 'Mostrar',
                total: 'Total',
                visible: 'Visivel',
                hidden: 'Escondido',
                enabled: 'Ativado',
                disabled: 'Desativado',
                auto: 'Automático',
                manual: 'Manual',
                all: 'Todos',
                locked: 'Bloqueado',
                complete: 'Completo',
                incomplete: 'Incompleto',
                games: 'Jogos',
                bundles: 'Pacotes',
                addons: 'Suplementos',
                wishlist: 'Lista de Desejos',
                responsive: 'Responsivo',
                windowed: 'Modo Janela',
                fullscreen: 'Ecrã Completo',
                onsale: 'Em Promoção',
                prodeals: 'Promoções Pro',
                userprofile: 'Meu Perfil',
                usermedia: 'Capturas e Vídeos',
                searchbtnbase: 'Pesquisar em',
                avatarpopup: 'Novo URL para avatar (vazio para o padrão):',
                sessiontime: 'Tempo de sessão',
                codec: 'Codec',
                resolution: 'Resolução',
                hardware: 'Hardware',
                software: 'Software',
                trafficsession: 'Tráfego da sessão',
                trafficcurrent: 'Tráfego atual',
                trafficaverage: 'Tráfego médio',
                packetloss: 'Pacotes perdidos',
                framedrop: 'Frames perdidos',
                latency: 'Latencia',
                jitter: 'Buffer de Jitter',
                decodetime: 'Tempo de descodificação',
                compression: 'Compressão',
                streammon: 'Monitor de Streaming',
                stream: 'Stream',
                community: 'Comunidade',
                speedtest: 'Teste de velocidade',
                quickaccess: 'Acesso Rápido',
                messages: 'Mensagens',
                comfeature: 'Ferramentas da Comunidade',
                avatar: 'Avatar',
                interface: 'Interface',
                shortcut: 'StadiaIcons',
                shortcuttitle: 'Instalar atalho para',
                shortcutdesc: 'Permite-te instalar um atalho para um jogo no teu dispositivo',
                stadiastats: 'StadiaStats',
                stadiastatsopen: 'Ver em StadiaStats.gg',
                stadiastatsdesc: 'Ativa atalhos para estatísticas de jogos, link para o perfil e um sistema para encontrar amigos em StadiaStats.gg',
                gridsize: 'Tamanho da Grelha',
                griddesc: 'Muda a quantidade de jogos em cada linha na biblioteca',
                clock: 'Relógio',
                clockdesc: 'Mostra a hora atual na lista de amigos, em sobreposição no jogo, ou ambos.',
                friendslist: 'Lista de Amigos',
                igoverlay: 'Sobreposição no jogo',
                listoverlay: 'Lista e Sobreposição',
                filter: 'Filtro de Jogos',
                filterdesc: 'Permite organizar a biblioteca de jogos, escondendo jogos. O filtro pode ser ativado/desativado através do símbolo, no canto superior direito acima dos jogos na biblioteca.',
                invitebase: 'Copiar ligação de Convite',
                inviteactive: 'Copiado!',
                gamelabel: 'Etiquetas de Jogos',
                gamelabeldesc: 'Remove as Etiquetas como por exemplo "Pro" dos jogos da página inicial.',
                homegallery: 'Galeria do Utilizador',
                homegallerydesc: 'Esconde a área "Capturas" no fundo do ecrã inicial.',
                quickprev: 'Pré-visualização de mensagens',
                quickprevdesc: 'Esconde a pré-visualização de mensagens na tua lista de amigos.',
                quickrep: 'Resposta Rápida',
                quickrepdesc: 'Esconde a opção de resposta rápida nas conversas.',
                offlinefriend: 'Amigos Offline',
                offlinefrienddesc: 'Esconde amigos offline na lista de amigos.',
                invisiblefriend: 'Amigos Invisíveis',
                invisiblefrienddesc: 'Esconde amigos com estado desconhecido na lista de amigos.',
                streammode: 'Modo de Streaming',
                streammodedesc: 'Torna certos elementos (p.e. a lista de amigos) não legíveis enquanto estás a fazer streaming (via ferramentas como OBS ou Discord).',
                catprev: 'Pré-visualização da Categoria',
                catprevdesc: 'Esconde a etiqueta da categoria ao passar por cima de um jogo.',
                streammondesc: 'Ativa para iniciar o Monitor quando um jogo começa.',
                resolutiondesc: 'A resolução pretendida para stream de jogos. (1440p e 2160p utilizam o Codec VP9)',
                codecdesc: 'O Codec utilizado para stream de jogos.',
                confirmreset: 'De certeza que queres reiniciar as configurações?',
                gamesfinished: 'Jogo Terminado',
                achievementsunlocked: 'Conquistas Desbloqueadas',
                totalPlayTime: 'Tempo Total de Jogo',
                splitstore: 'Dividir Listas da Loja',
                splitstoredesc: 'Divide as listas da loja em duas colunas para uma melhor visão geral.',
                inlineimage: 'Prévia de imagem',
                inlinedesc: 'Substitui links de imagem para formatos comuns de ficheiros (jpg/gif/png) com uma prévia clicável.',
                familyelements: 'Opções de partilha de família',
                familyelementsdesc: 'Esconder a opção "Partilhar este jogo com a família."',
                resetsettings: 'Reiniciar Configurações'

**Reply by ChristopherKlay** [reply]
Will be included in `1.11.3` (:

**Comment by Gabsletobar** [comment]


**Reply by Gabsletobar** [reply]
ps: There's no entry to translate "Statistics" in the profile.

![stadia_stats_profile](https://user-images.githubusercontent.com/31623861/124581123-b7f17b80-de48-11eb-9b53-bbf7fd61bb45.png)

**Reply by Gabsletobar** [reply]
**V1.14.1**

**Comment by Gabsletobar** [comment]
default: 'Padrão',
                native: 'Nativo',
                hide: 'Esconder',
                show: 'Mostrar',
                total: 'Total',
                visible: 'Visível',
                hidden: 'Escondido',
                enabled: 'Ativado',
                disabled: 'Desativado',
                auto: 'Automático',
                manual: 'Manual',
                all: 'Todos',
                locked: 'Bloqueado',
                complete: 'Completo',
                incomplete: 'Incompleto',
                games: 'Jogos',
                allgames: 'Todos os Jogos',
                leavepro: 'A sair do PRO',
                bundles: 'Pacotes',
                addons: 'Suplementos',
                wishlist: 'Lista de Desejos',
                responsive: 'Responsivo',
                windowed: 'Modo Janela',
                fullscreen: 'Ecrã Completo',
                onsale: 'Em Promoção',
                prodeals: 'Promoções PRO',
                userprofile: 'Meu Perfil',
                usermedia: 'Capturas e Vídeos',
                searchbtnbase: 'Pesquisar em',
                avatarpopup: 'Novo URL para avatar (vazio para o padrão):',
                sessiontime: 'Tempo da Sessão',
                codec: 'Codec',
                resolution: 'Resolução',
                hardware: 'Hardware',
                software: 'Software',
                trafficsession: 'Tráfego da Sessão',
                trafficcurrent: 'Tráfego Atual',
                trafficaverage: 'Tráfego Médio',
                packetloss: 'Pacotes Perdidos',
                framedrop: 'Frames Perdidos',
                latency: 'Latência',
                jitter: 'Buffer de Jitter',
                decodetime: 'Tempo de Descodificação',
                compression: 'Compressão',
                bitrate: 'Bitrate',
                streammon: 'Monitor do Streaming',
                stream: 'Stream',
                network: 'Internet',
                session: 'Sessão',
                extdetail: 'Mais Detalhes',
                maxresolution: 'Resolução Máxima',
                fps4K: 'Fotogramas a 4K',
                datadiscl: 'Esta é a taxa máxima de fotogramas alcançada ao jogar um jogo em modo 4K (deve ser um subscritor PRO). \
                Nos jogos com opções de resolução/taxa de fotogramas, a resolução foi escolhida. \
                Estes dados são fornecidos por <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
                e a base de dados completa pode ser encontrada <a href="https://airtable.com/shr32bmiOThVvSGar/tblAeJTnP2bzZyews" target="_blank">aqui</a>.',
                noteOne: 'Especificações confirmadas pelos desenvolvedores',
                noteTwo: 'Contador de Pixel',
                noteThree: '60FPS em 1080P',
                noteFour: '30FPS em 1080P',
                noteFive: 'Opção Performance/Qualidade',
                noteSix: 'Sem opções HDR',
                noteSeven: 'Não é compatível com 4K',
                community: 'Comunidade',
                speedtest: 'Teste de Velocidade',
                quickaccess: 'Acesso Rápido',
                messages: 'Mensagens',
                comfeature: 'Ferramentas',
                avatar: 'Avatar',
                interface: 'Interface',
                shortcut: 'StadiaIcons',
                shortcuttitle: 'Instalar atalho para',
                shortcutdesc: 'Permite-te instalar um atalho para um jogo no teu dispositivo',
                stadiastats: 'StadiaStats',
                stadiastatsopen: 'Ver em StadiaStats.gg',
                stadiastatsdesc: 'Ativa atalhos para estatísticas de jogos, link para o perfil e um sistema para encontrar amigos em StadiaStats.gg',
                gridsize: 'Tamanho da Grelha',
                griddesc: 'Muda a quantidade de jogos em cada linha na biblioteca',
                clock: 'Relógio',
                clockdesc: 'Mostra a hora atual na lista de amigos, em sobreposição no jogo, ou ambos.',
                friendslist: 'Lista de Amigos',
                igoverlay: 'Sobreposição no Jogo',
                listoverlay: 'Lista e Sobreposição',
                filter: 'Filtro de Jogos',
                filterdesc: 'Permite organizar a biblioteca de jogos, escondendo jogos. O filtro pode ser ativado/desativado através do símbolo, no canto superior direito acima dos jogos na biblioteca.',
                invitebase: 'Copiar ligação de Convite',
                inviteactive: 'Copiado!',
                gamelabel: 'Etiquetas de Jogos',
                gamelabeldesc: 'Remove as Etiquetas como por exemplo "Pro" dos jogos da página inicial.',
                homegallery: 'Galeria do Utilizador',
                homegallerydesc: 'Esconde a área "Capturas" no fundo do ecrã inicial.',
                quickprev: 'Pré-visualização de Mensagens',
                quickprevdesc: 'Esconde a pré-visualização de mensagens na tua lista de amigos.',
                quickrep: 'Resposta Rápida',
                quickrepdesc: 'Esconde a opção de resposta rápida nas conversas.',
                offlinefriend: 'Amigos Offline',
                offlinefrienddesc: 'Esconde amigos offline na lista de amigos.',
                invisiblefriend: 'Amigos Invisíveis',
                invisiblefrienddesc: 'Esconde amigos com estado desconhecido na lista de amigos.',
                notification: 'Notificações',
                notificationdesc: 'Apresentar uma notificação quando o Stadia Enhanced é atualizado para uma nova versão ("Auto" esconde após 5 segundos, "Manual" permanece até à interação do utilizador).',
                streammode: 'Modo de Streaming',
                streammodedesc: 'Torna certos elementos (p.e. a lista de amigos) não legíveis enquanto estás a fazer streaming (via ferramentas como OBS ou Discord).',
                catprev: 'Pré-visualização da Categoria',
                catprevdesc: 'Esconde a etiqueta da categoria ao passar por cima de um jogo.',
                streammondesc: 'Ativa para iniciar o Monitor quando um jogo começa.',
                resolutiondesc: 'A resolução pretendida para stream de jogos. (1440p e 2160p utilizam o Codec VP9)',
                codecdesc: 'O Codec utilizado para stream de jogos.',
                confirmreset: 'De certeza que queres reiniciar as configurações?',
                gamesfinished: 'Jogos Terminados',
                achievementsunlocked: 'Conquistas Desbloqueadas',
                totalPlayTime: 'Tempo Total de Jogo',
                splitstore: 'Dividir Listas da Loja',
                splitstoredesc: 'Divide as listas da loja em duas colunas para uma melhor visão geral.',
                inlineimage: 'Prévia de Imagem',
                inlinedesc: 'Substitui links de imagem para formatos comuns de ficheiros (jpg/gif/png) com uma prévia clicável.',
                familyelements: 'Opções de Partilha de Família',
                familyelementsdesc: 'Esconder a opção "Partilhar este jogo com a família."',
                donations: 'Doações',
                reportbug: 'Reportar problemas da extensão Stadia Enhanced',
                resetsettings: 'Reiniciar Configurações',

**Reply by Gabsletobar** [reply]
**Updated for V1.14.1**
There's also no way to select Portuguese on the  Language drop menu.

**Thanks!**

**Reply by ChristopherKlay** [reply]
Included the updated translation into the dev-build for `1.15` which will come after some more tests on the discord presence part.

About the language dropdown however; What isn't working correctly for you?
![image](https://user-images.githubusercontent.com/12472611/124638028-c82a4a80-de8a-11eb-8d58-44dadb3fdb30.png)
If you are not on a account that is set to Portuguese (in the settings of the Google account), the language is selectable. If you are, you can simply click the very first option to default to it - which always switches to the language your account is on.

It might be a bit confusing at first, but "default" is always your accounts language. (:

**Reply by Gabsletobar** [reply]
Oh ok, so it should be working fine the language menu since my google account is Portuguese by default. Sorry and thanks for the explanation.

Aside of that it appears that there's no entry to translate the word "Statistics" on the profile page. Image:
![image](https://user-images.githubusercontent.com/31623861/124639711-6535b500-de84-11eb-857b-52ac8ba08e80.png)

**Reply by ChristopherKlay** [reply]
Yea, i missed that when converting the layout into the extension. I already added it to the translation and it will be part of `1.15` as well. What would it be in portuguese? I'll directly include it (:

**Reply by Gabsletobar** [reply]
"What would it be in portuguese?"

Estatísticas

**Reply by ChristopherKlay** [reply]
Included. Will show up with `1.15` hopefully before the weekend (:

**Reply by Gabsletobar** [reply]
Thanks ;)
